### PR TITLE
Saleor 2551/allow assigning channels to shipping zones

### PR DESCRIPTION
**Pull request by mmarkusik**
I want to merge this change because it adds ability to assign channels to shipping zones.

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [x] This code contains UI changes
2. [x] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [x] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://feature-variants-and-shipping-zones-per-channel.api.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-2551 Allow assigning channels to shipping zones](https://app.clickup.com/t/2549495/SALEOR-2551)

**Comment by patrys** [comment]
Task linked: [SALEOR-2552 Allow assigning shipping zones to channels](https://app.clickup.com/t/2549495/SALEOR-2552)

**Review by orzechdev — APPROVED** [review]

File: src/shipping/components/ShippingZoneDetailsPage/utils.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { ShippingZone_shippingZone } from "@saleor/shipping/types/ShippingZone";
+import { mapMetadataItemToInput } from "@saleor/utils/maps";
+
+import { FormData } from "./types";
+
+export const getInitialFormData = (
```
Comment (dominik-zeglen) [inline src/shipping/components/ShippingZoneDetailsPage/utils.ts:6]: :+1:

File: src/shipping/components/ShippingZoneSettingsCard/ShippingZoneSettingsCard.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import { Channels_channels } from "@saleor/channels/types/Channels";
+import CentralPlacementDecorator from "@saleor/storybook/CentralPlacementDecorator";
+import CommonDecorator from "@saleor/storybook/Decorator";
+import { storiesOf } from "@storybook/react";
+import React from "react";
+
+import { FormData } from "../../components/ShippingZoneDetailsPage/types";
+import ShippingZoneSettingsCard from "./ShippingZoneSettingsCard";
+
+const props = {
```
Comment (dominik-zeglen) [inline src/shipping/components/ShippingZoneSettingsCard/ShippingZoneSettingsCard.stories.tsx:10]: Can we type it instead of its fields?

**Review by dominik-zeglen — CHANGES_REQUESTED** [review]

**Comment by maarcingebala** [comment]
@jwm0 @mmarkusik I suggest that we first merge this PR to the feature branch and then we'll do the QA, so that we have a full feature working on the feature branch.

**Pull request merged**